docs/components/dropdown.md

```
@@ -239,6 +240,25 @@ import { FwbDropdown } from 'flowbite-vue'
+  <fwb-dropdown text="Normal state">
+    Access this content by clicking the dropdown's trigger button
+  </fwb-dropdown>
+  <fwb-dropdown text="Disabled state" disabled>
+    You cannot access this content, since the dropdown's trigger button is disabled
+  </fwb-dropdown>
+</template>
+
+<script setup>
+import { FwbDropdown } from 'flowbite-vue'
+</script>
+```
```

Markdownlint (0.37.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)